### DISCUSSION TITLE
Remove the word "documentation" and the version number from the top left corner of the documentation

### DISCUSSION THREAD

**Discussion by rozsasarpi** [pradyunsg/furo#202] [Potential Issue] [answered]
First, thanks for the great package!

I cannot find a way to have only the name of the project in the top left corner of the generated html page. I get the version number and the word "documentation" added as well:

![image](https://user-images.githubusercontent.com/6670894/126864918-39bcd501-83c4-44d4-9dd9-5ab5a5500929.png)

Here is the live documentation and the `conf.py` file:

https://lightkde.readthedocs.io/en/stable/

https://github.com/rozsasarpi/lightkde/blob/a65d5121c951a7fc4d69de8aab9a3af456078a66/docs/conf.py

If I use the default `sphinx` theme instead of `furo`, I have only the name of the project (desired behavior). In another project where I use `furo`, I do not have the above described issue, so I probably miss something obvious and/or it might be related to versions.

**Comment by rozsasarpi** [comment, marked as answer]
Oh, I just found the solution in [`furo`'s `conf.py` file](https://github.com/pradyunsg/furo/blob/900ee03d816b2dcfd9b1a4fd3fb9630a5ad72d66/docs/conf.py#L76):

```
html_title = "your desired title"
```

This option could be added to the documentation.

**Reply by pradyunsg** [reply]
Glad you figured it out.

Do you want to file a PR adding a new page titled "Changing sidebar title" in the "Customisation" section, using a writing style as https://pradyunsg.me/furo/customisation/landing-page/? :)

**Reply by rozsasarpi** [reply]
Sure, I will submit a PR in the coming days.
